From 3ad386c4e2386a85052edca9dfd261e122a2b026 Mon Sep 17 00:00:00 2001
From: Rajesh Bhagat <rajesh.bhagat@freescale.com>
Date: Thu, 14 Jan 2016 10:58:59 +0530
Subject: [PATCH 013/151] usb: kconfig: remove dependency FSL_SOC for ehci fsl
 driver

CONFIG_USB_EHCI_FSL is not dependent on FSL_SOC, it can be built on
non-PPC platforms.

Signed-off-by: Rajesh Bhagat <rajesh.bhagat@nxp.com>
---
 drivers/usb/host/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/Kconfig b/drivers/usb/host/Kconfig
index 688698a..5e0ae5c 100644
--- a/drivers/usb/host/Kconfig
+++ b/drivers/usb/host/Kconfig
@@ -145,7 +145,7 @@ config XPS_USB_HCD_XILINX
 
 config USB_EHCI_FSL
 	bool "Support for Freescale PPC on-chip EHCI USB controller"
-	depends on FSL_SOC
+	depends on USB_EHCI_HCD
 	select USB_EHCI_ROOT_HUB_TT
 	select USB_FSL_MPH_DR_OF if OF
 	---help---
-- 
2.1.0.27.g96db324

